resources/jaas/kafka_server_jaas.conf

```diff
@@ -0,0 +1,8 @@
+KafkaServer {
+  org.apache.kafka.common.security.plain.PlainLoginModule required
+  username="admin"
+  password="admin-secret"
+  user_admin="admin-secret"
```

Comment:
These necessary? Are they testing another way to configure user/pass?

Reply:
I looked around and this seemed to be the consistent way of configuring SASL for Kafka

Reply:
I looked around and this seems to be the recommended way to configure SASL.

Reply:
I looked around and this seems to be the recommended way to configure SASL for Kafka